docs-source/spring/content/platform/config/_index.md

```diff
@@ -1,32 +1,395 @@
 ---
-title: "Configuration"
+title: "Spring Config Server"
 ---
 
-Oracle Backend for Spring Boot and Microservices includes Spring Cloud Config which provides server- and client-side support for externalized
-configurations in a distributed system. The Spring Cloud Config server provides a central place to manage external properties for applications
-across all environments.
+Oracle Backend for Spring Boot and Microservices includes Spring Cloud Config which provides server- and client-side support for externalized configurations in a distributed system. The Spring Cloud Config server provides a central place to manage external properties for applications across all environments.
 
 The Spring Cloud Config server is pre-configured to work with the Spring Boot Eureka service registry, and it is configured to store the
-configuration in the Oracle Autonomous Database, so it easily supports labelled versions of configuration
-environments, as well as being accessible to a wide range of tools for managing the content.
-Configuration is stored in the `CONFIGSERVER` schema in the `PROPERTIES` table.
+configuration in the Oracle Autonomous Database, so it easily supports labeled versions of configuration environments, as well as being accessible to a wide range of tools for managing the content. More details can be found here: ([Spring Cloud Config Documentation](https://spring.io/projects/spring-cloud-config)). 
```

Comment:
Change to:

Configuration in the Oracle Autonomous Database to support labeled versions of configuration environments as well as ...

Reply:
Fixed

Reply:
Done